ledger/sync/src/scp_network_state.rs

```diff
@@ -58,12 +51,6 @@ impl<ID: GenericNodeId + Clone + Eq + PartialEq + Hash> SCPNetworkState<ID> {
         let entry_exists = self.id_to_current_slot.contains_key(&sender_id);
         let entry_is_older = entry_exists && self.id_to_current_slot[&sender_id] < new_slot_index;
         if !entry_exists || entry_is_older {
-            log::trace!(
```

Comment:
This is the only place that SCPNetworkState used its logger. Seemed like things would be simpler if I just removed the logger.